storybook/index.html

```diff
@@ -338,6 +338,20 @@ <h1 class="sectionHeading">BAMP</h1>
                         </div>
                     </center>
                 </section>
+                <section class="demo" data-module-name="pressModule">
+                    <div class="box box-red">
+                        <div class="pressModule">
+                            <img class="pressModule--img" src="https://i.ytimg.com/vi/f_oKFLeZi88/maxresdefault.jpg" alt="">
+                            <div class="pressModule--header">D4C Funny Valentine
+                                <div class="pressModule--text">Dirty Deeds Done Dirt Cheap allows different parallel worlds/dimensions to co-exist at the same time and/or in the same place</div>
```

Comment:
I think that you should add `<p class="...">` to make the text formatted.

Btw, nice work. I thought the pic was hillarious

Reply:
Some of the indents could be cleaned up too.

Reply:
another reason for using a `<p>` is that it *semantically* means that there's a paragraph there, but divs are just a generic element.

https://www.lifewire.com/why-use-semantic-html-3468271

(won't break anything, it's mostly about clarity in the code. but it does mean we could add `line-height` to `p` in `base.css` and it'd get applied here.)